packages/shopify-codemod/transforms/global-assignment-to-default-export.js

```diff
@@ -12,7 +12,7 @@ export default function globalAssignmentToDefaultExport({source}, {jscodeshift:
   }
 
   function isDefaultExportMemberAssignment(defaultExportName, assignmentTargetString) {
-    return new RegExp(`^${defaultExportName}\..+`).test(assignmentTargetString);
+    return new RegExp(`^${defaultExportName}\\..+`).test(assignmentTargetString);
```

Comment:
This looks useful, but should probably be in a separate PR.  It's easier to update CHANGELOG if we can just scan through commit messages.